apps/examples/smartfs/smartfs_powercut/smartfs_powercut_main.c

```diff
@@ -530,12 +540,12 @@ int init_test_file(void)
 	}
 
 	/* Create small random character files */
-	ret = create_file(TEST_SMALL_R, TEST_SMALL_BUFSIZE, FILE_TYPE_RANDOM);
+	ret = create_file(TEST_SMALL_R, TEST_SMALL_BUFSIZE, FILE_TYPE_RANDOM, false);
```

Comment:
Instead of modify create file itself, I think we'd better check all backup file's validation in this function, and if invalid then unlink.
That will be more clear role of create_file, I think.

Reply:
How about we rename "create_file" function to "init_file"
So that init_file function has both validation and creation parts both?
I think this way, we can make our changes with minimum duplication of code.
Source and backup files, both have to be validated during initialization and then created accordingly.

Reply:
We can also rename "init_test_files" to "initialize_test" to differentiate between the two better?

Reply:
Updated